contracts/IStorageManager.sol

```diff
@@ -34,13 +34,20 @@ interface IMiningHash {
 
 interface IDaggerHash {
     // Decode the data and return raw data that verfier needs
-    function unmaskWithEthash(
-        uint256 kvIdx,
-        bytes memory maskedData
+    function unmaskChunkWithEthash(
+        uint64 chunkIdx,
+        bytes32 kvHash,
+        address miner,
+        bytes memory maskedChunk
     ) external view returns (bytes memory);
 
+
     // Another option is to use Verify Delay Function(VDF),
     // such as MiMC. For saving gas cost, those are not materialized for now
+    function unmaskWithEthash(
+        uint256 kvIdx,
+        bytes memory maskedData
+    ) external view returns (bytes memory);
```

Comment:
Oh we don't have to keep this if no implementation being put. Previously I want to keep an interface for all options. Leaving a message should be good. 

Real codes cost extra gas.

Reply:
But the _hashimotoKeccak256() and _hashimoto() still use this function, I think we can keep them for a while.